src/Compilers/CSharp/Test/Emit3/Semantics/ExtensionTests2.cs

```diff
@@ -14471,15 +14469,34 @@ static void Test()
 """;
 
         var comp2 = CreateCompilation(src2);
-        comp2.VerifyDiagnostics(
-            // (15,9): error CS0131: The left-hand side of an assignment must be a variable, property or indexer
-            //         default(S1)[0] += 1;
-            Diagnostic(ErrorCode.ERR_AssgLvalueExpected, "default(S1)[0]").WithLocation(15, 9)
-            );
+        if (refKind == "ref readonly")
+        {
+            comp2.VerifyDiagnostics(
+                // (15,9): warning CS9193: Argument 0 should be a variable because it is passed to a 'ref readonly' parameter
```

Comment:
>// (15,9): warning CS9193: Argument 0 should be a variable because it is passed to a 'ref readonly' parameter [](http://example.com/codeflow?start=16&length=109)

Do we report this warning for an extension property? #Closed

Reply:
It looks like we do